org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/ProjectManager.xtend

```diff
@@ -42,7 +41,6 @@ class ProjectManager {
     @Inject protected IncrementalBuilder incrementalBuilder
     @Inject protected Provider<XtextResourceSet> resourceSetProvider
     @Inject protected IResourceServiceProvider.Registry languagesRegistry
-    @Inject protected IFileSystemScanner fileSystemScanner
```

Comment:
The removal is a breaking API change. Can we leave it there even if it is unused?